column.go

```diff
@@ -101,7 +101,6 @@ func NewColumn(h api.SQLHSTMT, idx int) (Column, error) {
 	default:
 		return nil, fmt.Errorf("unsupported column type %d", sqltype)
 	}
-	panic("unreachable")
```

Comment:
This requirement changed in Go 1.1 https://golang.org/doc/go1.1#return, which was released 2013-05-13. It's now a `go vet` warning to have unreachable code: `column.go:104: unreachable code`.

If you'd like `odbc` to remain compilable with Go 1.0 you could move the default return outside of the switch, which would also remove the `go vet` warning. Although I don't have a way to guarantee overall compatibility with Go 1.0 without compiling Go from source, the oldest version available for download is Go 1.2.2.

Please let me know what you'd like to do and I'll make the update.